### PR TITLE
Push copied code from Trials Project

### PR DESCRIPTION
**Pull request by myelin0**
Added copied trial project, "Technical Page."
-html file
-css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        <nav id="navbar">
+            
+            <header >JS Documentation</header>
+            <a href="#Introduction" class="nav-link">Introduction</a>
+            <a href="#What_you_should_already_know" class="nav-link">What you should already know</a>
+            <a href="#JavaScript_and_Java" class="nav-link">JavaScript and Java</a>
+            <a href="#Hello_world" class="nav-link">Hello world</a>
+            <a href="#Declaring_variables" class="nav-link">Declaring variables</a>
+            
+        </nav>
```
Comment (myelin0) [inline index.html:20]: When I click on a navbar element, the page should navigate to the corresponding section of the main-doc element. Great job!! 🥇 😎 👍 
I would suggest though, putting the anchor links of the navbar as list items `li` of a `ul`.
It is generally a good practice to use list items for the navigation bar elements.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        <main id="main-doc">
+            
+            <section id="Introduction" class="main-section">
+                <header> <h1>Introduction</h1></header>
+                <p>JavaScript is a cross-platform, object-oriented scripting
```
Comment (myelin0) [inline index.html:25]: You have used the right tag for the `header`, 👏 💯   however, if you want to use semantic tags throughout your code, you could wrap your `p` tags inside an `article` tag. Check out the [HTML Semantic Elements from w3schools] (https://www.w3schools.com/html/html5_semantic_elements.asp) for reference.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+            </section>
+            <section id="Hello_world" class="main-section">
+                <header > <h4>Hello world</h4></header>
+                <p>To get started with writing JavaScript, open the Scratchpad
+                    and write your first "Hello world" JavaScript code:
+                </p>
+                <code>
+                    function greetMe(yourName) { alert("Hello " + yourName); }
+                    greetMe("World");
+                </code>
```
Comment (myelin0) [inline index.html:131]: When inserting <`code`> tags, you could use a different background color so it looks better, like so :

code-background

File: README.md [hunk 4]
```diff
@@ -1 +1 @@
-# review-trials
\ No newline at end of file
+# projectWithAdama
```
Comment (myelin0) [inline README.md:1]: A well-written, descriptive ReadMe page introduces your project to potential employers and makes you look professional. 😎  I suggest you use  [this template](https://github.com/microverseinc/readme-template) , but remember to customize it to your project.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    font-weight:bold;
+}
+
+
+
+
+
+
+
+
```
Comment (myelin0) [inline style.css:17]: It seems you have empty spaces in your style file. It is best to use Linters to correct that, so you have a clean code. 😉

**Review by myelin0 — COMMENTED** [review]
Good job so far!

HIGHLIGHTS!
Good usage of relative links of the navbar.💯
There are some issues that you still need to work on to go to the next project but you are 
E4u7V_YXEAIiFhR


Required Changes ♻️
Check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Review by Cathella — APPROVED** [review]
## 🟢 STATUS: APPROVED

### Hi @myelin0 ✋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## HIGHLIGHTS 👍🏻 

✔️ Utilized the "Review changes" feature from GitHub.
✔️ Phrased the comments politely, as suggestions
✔️ Always includes an explanation, a "why", for each requested change.
✔️ Comments are clear and easy to understand

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**